src/components/login/SignUpModal.tsx

```
@@ -19,166 +22,189 @@ export default function SignupModal({ isOpen, onClose }: SignupModalProps) {
+      setConfirmPassword(e.target.value)
+    }
+  }
 
   // 회원가입 로직
   const handleSubmit = (e: React.FormEvent) => {
     e.preventDefault()
-    apiCall("/api/user", "POST", { nickname, email, password }).then(({ data }: { data: any }) => {
-      console.log(data)
-      onClose()
+    apiCall("/api/user", "POST", { nickname, email, password }).then(({ error }: { data: any, error: string | null }) => {
+      if (error) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)